app/cp_nigeria/views.py

```diff
@@ -1470,20 +1470,13 @@ def cpn_kpi_results(request, proj_id=None):
                 factor = 100.0
                 unit = "%"
                 # TODO quick fix for renewable share, fix properly later (this also doesnt include possible renewable share from grid)
-                if options.community is not None:
-                    scen_values = [
-                        round(
-                            inverter_flow / np.sum(get_aggregated_demand(community=options.community)) * factor,
-                            2,
-                        )
-                    ]
-                else:
-                    scen_values = [
-                        round(
-                            inverter_flow / np.sum(get_aggregated_demand(proj_id=proj_id)) * factor,
-                            2,
-                        )
-                    ]
+
+                scen_values = [
+                    round(
+                        inverter_flow / np.sum(get_aggregated_demand(project)) * factor,
```

Comment:
actually get_aggregated_demand should only be used once to compute the aggregated demand from the consumer groups, then we should simply get the value stored in the demand instead of recalculating the aggregated demand. for now it won't affect fonctionallty